.github/ISSUE_TEMPLATE/scraper_bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Scraper bug report
+about: Report a scraper that is not working correctly
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+Thanks for filing a bug report with us!  To help get the issue fixed, please fill in the information below.
+
+**The URL of the recipe(s) that are not being scraped correctly**
+
+- https:// ...
+
+**The version of the `recipe-scrapers` software you're using**
```

Comment:
I'm not sure what our major mode of operation should be on this. 

On one hand we're not backporting fixes to previous versions, and we've only had one major breaking change with Python versions (#188 ) so "update to latest version" might be the best way of ensuring consistency. Updating the version might be something to put on the "do this before filing bug" procedure.

On the other most of our business logic doesn't change with version updates. A bug in version 8 is probably still around in version 10 etc.

Reply:
That's a pretty good idea, I think.  There's definitely ambiguity in whether the author would expect a fix to land for their version (without any other changes), and whether there's any duty for them to try to check for bugfixes and/or verify in the latest version (which we probably should, given that it's relatively quick and can save everyone a lot of time).

I'll apply some changes here soon.